core/commands/list_cloud_instances_command.rb

```diff
@@ -67,9 +69,30 @@ def generate_gcp_list
     end
     return Result.error('No instances were found in GCP-services') if all_instances.empty?
 
+    all_instances.each do |instance|
+      instance[:launch_time] = translate_to_right_time(instance[:launch_time])
+    end
+    select_by_time(all_instances) unless @env.hours.nil?
     Result.ok(all_instances)
   end
 
+  def translate_to_right_time(time)
+    date, times = time.split('T')
```

Comment:
Use Ruby DateTime class from build in date library, https://ruby-doc.org/stdlib-2.7.1/libdoc/date/rdoc/DateTime.html instead of manually parsing the time string.